test/ode/ode_low_storage_rk_tests.jl

```diff
@@ -312,6 +312,31 @@ sol_new = solve(new_prob_ode_nonlinear_inplace, alg, dt=1.e-4, save_everystep=fa
 @test sol_old[end] ≈ sol_new[end]
 
 
+alg = CKLLDDRK43_2()
+dts = 1 ./ 2 .^(8:-1:4)
+for prob in test_problems_only_time
+  sim = test_convergence(dts, prob, alg)
+  @test sim.𝒪est[:final] ≈ OrdinaryDiffEq.alg_order(alg) atol=testTol
+end
+for prob in test_problems_linear
+  sim = test_convergence(dts, prob, alg)
+  @test_broken sim.𝒪est[:final] ≈ OrdinaryDiffEq.alg_order(alg) atol=testTol
+end
+for prob in test_problems_nonlinear
+  sim = test_convergence(dts, prob, alg)
+  @test sim.𝒪est[:final] ≈ OrdinaryDiffEq.alg_order(alg) atol=testTol
+end
+integ = init(prob_ode_large, alg, dt=1.e-2, save_start=false, save_end=false, save_everystep=false)
+@test Base.summarysize(integ) ÷ Base.summarysize(u0_large) <= 7
+integ = init(prob_ode_large, alg, dt=1.e-2, save_start=false, save_end=false, save_everystep=false, alias_u0=true)
+@test Base.summarysize(integ) ÷ Base.summarysize(u0_large) <= 7
```

Comment:
@ranocha I was talking about these constants. I'm not able to figure out the optimum value

Reply:
- The schemes are 2R+ schemes, so they should have a storage of `2` for `adaptive==false` and `3` for `adaptive==true`. In order to enable less space for `adaptive==false`, you have to opt-in to this optimisation via  
 https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/d155eb474f5569fdfdcccb5e02583f9d28db86cb/src/alg_utils.jl#L416-L428
  This is done in 
 https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/d155eb474f5569fdfdcccb5e02583f9d28db86cb/src/solve.jl#L222-L228
  For these new algorithms, we have to change `uses_uprev(alg::OrdinaryDiffEqAlgorithm)` to `uses_uprev(alg::OrdinaryDiffEqAlgorithm, adaptive::Bool)`, I think.
- We have one additional storage because the storage optimised assignments of the form `u1 = f(t, u1)` are not implemented yet.
- If `alias_u0==false` (default), there is one additional storage place for the initial condition. However, setting `alias_u0=true` should reduce the required storage.

To sum up: It seems that the current implementation uses a bit more storage that necessary and we need to change some of the internal functions. I need to have a deeper look at the implementation in order to see whether there are some optimisations.

Reply:
@ranocha I made the changes you suggested,
```
@test Base.summarysize(integ) ÷ Base.summarysize(u0_large) <= 5
@test Base.summarysize(integ) ÷ Base.summarysize(u0_large) <= 4
```
Now the tests work for these parameters

Reply:
I wrote a new uses_uprev function. But after this function 
```
@test Base.summarysize(integ) ÷ Base.summarysize(u0_large) <= 6
@test Base.summarysize(integ) ÷ Base.summarysize(u0_large) <= 5
```
memory usage ratio is 6 and 5